WordPress/src/main/java/org/wordpress/android/ui/posts/mediauploadcompletionprocessors/BlockProcessor.java

```diff
@@ -112,6 +125,10 @@ String processBlock(String block) {
         return block;
     }
 
+    String processBlock(String block) {
+        return processBlock(block, false);
+    }
```

Comment:
This function is also used in unit tests. So, in order to avoid adding the `isSelfClosingTag` argument to all of them, I overloaded the function for convenience.